.travis.yml

```diff
@@ -1,10 +1,13 @@
 # Use Ubuntu 14.04
 dist: trusty
+# Can't use sudo: false because of travis-ci/travis-ci#7019
+sudo: required
```

Comment:
@dwijnand I tried oraclejdk8 in a separate branch, but it still fails for the same reason.

```
Error while processing /home/travis/build/ashawley/sbt-website/src/reference/01-General-Info/04-Bintray-For-Plugins.md
java.lang.StackOverflowError
	at java.util.regex.Pattern.group0(Pattern.java:2888)
	at java.util.regex.Pattern.sequence(Pattern.java:2051)
	at java.util.regex.Pattern.expr(Pattern.java:1996)
	at java.util.regex.Pattern.compile(Pattern.java:1696)
	at java.util.regex.Pattern.<init>(Pattern.java:1351)
	at java.util.regex.Pattern.compile(Pattern.java:1028)
```

https://travis-ci.org/ashawley/sbt-website/builds/232983632